src/Provider/Line.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/*!
+* Hybridauth
+* https://hybridauth.github.io | https://github.com/hybridauth/hybridauth
+*  (c) 2017 Hybridauth authors | https://hybridauth.github.io/license.html
```

Comment:
You might want to fix the year and add yourself as the author, for future reference.